lib/wickedbase.pm

```diff
@@ -538,6 +552,13 @@ sub wait_for_dhcpd {
     die('DHCP not comming up');
 }
 
+sub ifbind {
+    my ($self, $action, $interface) = @_;
+    my $cmd = "./ifbind.sh $action $interface";
```

Comment:
```suggestion
    my $cmd = WICKED_DATA_DIR . '/wicked/ifbind.sh $action $interface";
```

This could avoid https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/7645/files#diff-3732d4fecc2fdcf6cae9a8d0b7eaa823R56

Reply:
we would still need to make it executable (chmod command). So, we don't avoid any line

Reply:
ic, just in case, we could call it with 
```suggestion
     my $cmd = 'bash ' . WICKED_DATA_DIR . '/wicked/ifbind.sh $action $interface";
```